src/arrays/ArrayUtil.java

```diff
@@ -46,50 +67,78 @@ public static boolean contains(int[] array, int value) {
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
     public static int indexOf(int[] array, int value) {
-
-        return 0;
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == value)
+                return i;
+        }
+        return -1;
     }
 
     /**
      * Remove the element by the given index.
      * Print new array
      */
     public static void remove(int[] array, int index) {
-
+        if (index < 0)
+            System.out.println("Incorrect index");
```

Comment:
no need to continue to invoke this method if index < 0, just put  **return;**  after  **System.out.println("Incorrect index");**